src/sentry/models/releasefile.py

```diff
@@ -340,11 +344,11 @@ def _key_fields(self):
 
 
 def read_artifact_index(
-    release: Release, dist: Optional[Distribution], **filter_args
+    release: Release, dist: Optional[Distribution], use_cache: bool = False, **filter_args
```

Comment:
`default=False` because we don't want to cache it in endpoints, or is there another reason?

Reply:
That was my intention, yes. But we might want to cache it there as well at some point, if the necessity arises and the infrastructure can handle it.